doc/man3/EVP_PKEY_is_a.pod

```diff
@@ -28,6 +36,8 @@ otherwise 0.
 EVP_PKEY_can_sign() returns 1 if the I<pkey> key type functionality
 supports signing, otherwise 0.
 
+EVP_KEYMGMT_get0_first_name() returns the name that is found or NULL on error.
```

Comment:
Should be EVP_PKEY_get0_first_alg_name().

Reply:
Interestingly enough, I'm adding `EVP_PKEY_typenames_do_all()` in #12873...

Reply:
This trivial doc copy&paste error is still unresolved.